File: Exercicios_Em_Python_CV/desafio_012.py
#Faça um algoritmo que leia o preço de um produto
#e mostre seu novo preço, com %5 de desconto
preco_do_produto = float(input('Digite o preço do produto:'))
novo_preco = preco_do_produto * 0.95
print('O produto tem preco incial R${:.2f} e agora com desconto de %5 tera o valor de R${:.2f}'.format(preco_do_produto,novo_preco))

File: Exercicios_Em_Java_CV/Desafio012.java
package fundamentos;

import java.util.Scanner;

/*
 * Faça um algoritmo que leia o preço de um produto e mostre seu
 * novo preco com 5% de desconto
 */
public class Desafio012 {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		float preco = scanner.nextFloat();
		scanner.nextLine();
		System.out.printf("Preco do produto:R$ %.2f", preco);
		System.out.println();
		System.out.printf("Novo preco:R$%.2f", preco*0.95);
		scanner.close();
	}

}
